examples/generative/corrdiff/training/training_loop.py

```diff
@@ -71,7 +73,11 @@ def training_loop(
     patch_shape_y=448,
     patch_num=1,
     wandb_mode="disabled",
+    wandb_project="Modulus-Generative",
```

Comment:
why are we adding the wandb configs here? If you want to make these configurable, you'd also need to add them to the hydra config file and parse them.